textdb/textdb-sandbox/JinggangHelloWorld.java

```diff
@@ -1,6 +0,0 @@
-#include<iostream>
-public class JinggangHelloWorld{
-	public static void main(String[] args){
-		cout << "Hello World!" << endl;
```

Comment:
@kishore-narendran OK. Please create a new PR for this minor change, and add me and @JavierJia as reviewers.